transform/models/marts/repo_pushes_daily.sql

```diff
@@ -0,0 +1,9 @@
+select distinct
+  repo_id,
+  repo_name,
+  date_trunc('day', event_date) as date_day,
+  count(*) as push_count,
+  sum(count(*)) over (partition by repo_id order by date_trunc('day', event_date)) as cumul_push_count
```

Comment:
Small nit pick: I'd suggest using `order by date_day` to avoid repetition of the `date_trunc`.

Reply:
Similarly, you can do `sum(push_count)` to avoid repetition of `count(*)`.

Reply:
@edsioufi Updated SQL query; could you please review it?